### DISCUSSION TITLE
A popup for downloading mods that have dependencies (required and/or optional)

### DISCUSSION THREAD

**Discussion by woodiertexas** [modrinth/.github#58] [Feature requests]
There currently is no notice for downloading mods that have dependencies.

I'd like a popup listing the mod version's dependencies when you go to download said version.

**Comment by Laxystem** [comment]
*But*, allow us to "Never show again", and to "don't show again for this mod" (unless the dependencies have changed)

**Comment by ffuentesm** [comment]
I agree with this, but not just to show the dependencies when you download, but to have another tab on the mod page with "Dependencies" that show the link of the mod requirements. 
That would also be a helpful filter to filter mods by mod dependencies. 
So I could like on a "Create" or a "Thermal" or "Tinker's" dependents on the filter and it would display a list of all the mods that require those mods in the filter.

**Comment by T3sT3ro** [comment]
Or if it could open a wizard that has a selection list that shows which mods to download (required would be ticked automatically, optional would not) and it would start many files download or a download of a zip/tarball of selected mod, that would be cool and useful.
